### PR TITLE
[IoT.js] Merge common buildflags

### PR DESCRIPTION
**Pull request by bzsolt**
Some trailing whitespace are also removed.

JSRemoteTest-DCO-1.0-Signed-off-by: Zsolt Borbély zsborbely.u-szeged@partner.samsung.com

### CODE REVIEW

File: API/application/iotjs.py [hunk 1]
```diff
@@ -129,35 +128,34 @@ def build(self):
...
+        # is based on this target.
+        minimal_build_flags = list(common_build_flags)
+        minimal_build_flags.append('--target-board=rpi2')
+        minimal_build_flags.append('--builddir=%s' % paths.IOTJS_MINIMAL_BUILD_PATH)
+
+        # Run the buildscript with minimal build flags for binary information.
         utils.execute(paths.IOTJS_PATH, 'tools/build.py', minimal_build_flags)
 
         # The following builds are target specific with memory usage features.
+        build_flags = list(common_build_flags)
```
Comment (rtakacs) [inline API/application/iotjs.py]: Could we move this line below the `include_modules = ['spi', 'uart']` line? In this case `build_flags` could be in one place.

Comment (robertsipka) [reply API/application/iotjs.py]: LGTM

**Comment by robertsipka** [comment]
LGTM

File: API/application/iotjs.py (hunk 1 above)
Comment (bzsolt) [reply API/application/iotjs.py]: Fixed.

**Comment by rtakacs** [comment]
LGTM

**Pull request merged**